src/styles/blocks/card.scss

```
@@ -0,0 +1,124 @@
+@import 'variables';
+
+.container {
+  max-width: 1200px;
+  padding: $card-paddings;
+  justify-content: center;
+  display: grid;
+  grid-template-columns: repeat(auto-fit, $card-width);
+  gap: 46px 48px;
+}
+
+@each $breakpoint, $columns in $breakpoints {
+  @media (min-width: $breakpoint) {
+    .container {
+      grid-template-columns: repeat(#{$columns}, $card-width);
+    }
+  }
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: $card-width;
+  padding: 32px 16px 16px;
+  border: 1px solid $border-color;
+  border-radius: 5px;
+  background-color: $background-color;
+  font-family: $font-family;
+  transition: transform 0.3s ease;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__image {
+    object-fit: contain;
+    width: 166px;
+    height: 134px;
+    border-radius: 5px;
+  }
+
+  &__title {
+    text-align: left;
+    padding-top: 40px;
+    line-height: 18px;
+    font-size: 12px;
+    font-weight: 500;
+    color: $main-text-color;
+
+    &:hover {
+      color: #34568b;
+    }
+  }
+
+  &__productCode {
```

```suggestion
  &__product-code {
```
Usually in `html/css` we use kebab case for class names